packages/is-loki-running/src/index.native.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function isLokiRunning(win = global) {
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
This comment doesn't seem to apply to the line below

Reply:
Fixed in https://github.com/oblador/loki/commit/472067f089525ef163acc31dc00a0311bca9c4d5